### ISSUE TITLE
nif format suport

### ISSUE THREAD

**Issue by cj2000** [assimp/assimp#4790]
I know there is already a request for this feature, but it was closed, so I am asking again. There is already an API to work with this format.  The API is called niflib and here [the link](https://github.com/niftools/niflib?ysclid=l9znlsxafb893654197).

**Comment by cj2000** [comment]
Yes, the format can be really useful and like already mentioned there is an API to deal with that format, so some kind of integration of it into assimp can be a solution.

**Comment by tellypresence** [comment]
#3474 is tracking request to add support for NIF format, this should be closed
